.github/CODEOWNERS

```diff
@@ -1,3 +1,5 @@
 * @snyk/boost
 * @snyk/hammer
-template/iac/* @snyk/group-infrastructure-as-code
\ No newline at end of file
+template/iac/* @snyk/group-infrastructure-as-code
+template/code/* @snyk/nebula
```

Comment:
Please use `@snyk/sast-team` here for now, we have internal discussions where we will move ownership.